lib/screens/second_form_screen.dart

```diff
@@ -93,7 +100,7 @@ class _SecondFormScreenState extends State<SecondFormScreen> {
                 PrimaryButton(
                   text: 'Concluir',
                   onPressed: () {
-                    Navigator.pushNamed(context, '/form');
+                    Navigator.of(context).popUntil((route) => route.isFirst);
```

Comment:
Esse método basicamente fica dando pop até essa condição ser verdadeira?
Pensando um pouco no contexto da senha, seria legal se ela fosse um modal que impedisse o uso, mas pensando em fazer mais simples por causa do tempo (e não ter que lidar com a info de se o usuário já digitou a senha ou não), pensei em só fazer ela ser a primeira tela.
Por isso aqui eu deixaria algo tipo
```suggestion
                    Navigator.popUntil(context, ModalRoute.withName('/pagina_que_quisermos'));
```
Não sei se faz sentido pra você :D

Reply:
Mas caso dê tempo de fazer ela ser um modal como eu quero, aí é tranquilo dar popUntil a primeira mesmo 🤔